dockerfiles/models/Dockerfile

```diff
@@ -52,4 +51,8 @@ RUN ln -sf /opt/conda/bin/python3 /usr/local/bin/python3
 
 COPY ./dockerfiles/models/requirements.txt ./models-image-requirements.txt
 
-RUN python -m pip install -r models-image-requirements.txt
+# install rust since it's a dependency of tokenizers
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y && \
+    . $HOME/.cargo/env && \
+    rustc --version && \
+    python -m pip install -r models-image-requirements.txt
```

Comment:
do we really need this to be run together? How long it takes to install rust?

Reply:
it takes 10-20 seconds. you're right I'll split it

Reply:
so it is not as much as I thought.

the main question is whether we want to invalidate the cache for the rust installation when the `models-image-requirements.txt` gets updated... If yes because it also installs updates etc... so I can live with that, but if it only to have the rust binary then there is no need to have them in the same command.